config/routes.rb

```diff
@@ -1,8 +1,12 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
   resources :users, only: %i[index show] do
-    resources :posts, only: %i[index show]
+    resources :posts, only: %i[index new create show] do
+    end
```

Comment:
We should only embed our resources as follow: 
-> comments inside a post route, 
-> likes inside a post

Try something like this:
```suggestion
     resources :comments, only: %i[index new create show]     
     resources :likes, only: %i[create destroy]
 end
```

Reply:
## Hi @descholar-ceo 😃 

#### First of all thank you very much for the review. 👍 

I am going to nest the routes as you suggested me.

Kindly I would like to ask you, There will be no difficulty if I nest the routes more than one level deep?
I read a little about it at https://guides.rubyonrails.org/routing.html#nested-resources.

Thank you for your time.